mne_realtime/lsl_client.py

```diff
@@ -48,34 +48,42 @@ class LSLClient(_BaseClient):
     """
 
     @fill_doc
-    def get_data_as_epoch(self, n_samples=1024, picks=None):
-        """Return last n_samples from current time.
+    def get_data_as_epoch(self, n_samples=1024, picks=None, timeout=None):
+        """Return n_samples from LSL in FIFO order.
 
         Parameters
         ----------
         n_samples : int
             Number of samples to fetch.
         %(picks_all)s
+        timeout : Optional[float]
```

Comment:
Please use NumpyDoc format
```suggestion
        timeout : float | None
```